sdk/core/azure-core/src/private/package_version.hpp

```diff
@@ -24,13 +26,13 @@ namespace Azure { namespace Core { namespace _detail {
   class PackageVersion final {
   public:
     /// Major numeric identifier.
-    static constexpr int Major = AZURE_CORE_VERSION_MAJOR;
+    static constexpr int32_t Major = AZURE_CORE_VERSION_MAJOR;
```

Comment:
Didn't modify storage, but if we do this, we'd want to make the same change there.